Formula/clap-host.rb

```diff
@@ -0,0 +1,24 @@
+class ClapHost < Formula
+  desc "CLAP example host"
+  homepage "https://github.com/free-audio/clap-host"
+  url "https://github.com/free-audio/clap-host", using: :git, tag: "1.0", revision: "b5db1c246c173692371a17d92c2bd5fda1826d98"
```

Comment:
Do we need a Git clone? Can we use a tarball instead?

Reply:
It has submodules.
For now git clone is the way to go, maybe in the future we'll make tarball.
Is git clone problematic?